lib/Comment.php

```diff
@@ -472,24 +472,34 @@ protected function get_meta_values( $comment_id = null ) {
 	 * @api
 	 *
 	 * @param string $field_name The field name for which you want to get the value.
+     * @param array  $args       An array of arguments for getting the meta value. Third-party
+	 *                           integrations can use this argument to make their API arguments
+	 *                           available in Timber. Default empty.
 	 * @return mixed The meta field value.
 	 */
-	public function meta( $field_name ) {
+	public function meta( $field_name, $args = array() ) {
```

Comment:
I don't yet understand why we need this `args` argument. What is it for?

Reply:
@pascalknecht I'll answer this :) 

First of all look at https://github.com/timber/timber/issues/1827#issuecomment-431487553 - this is a list of plugins similar to ACF - some of them use more than one argument (even ACF have two arguments). Thanks to `$args` it will be easier to create integration for other plugins that will be using `meta`.